css-canvas-medical/styles.css

```diff
@@ -0,0 +1,192 @@
+* {
+  font-family: Avenir, sans-serif;
+}
+
+.container {
+  width: 1000px;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.col-full {
+  flex-basis: 100%;
+}
+
+.col-half {
+  flex-basis: 50%;
+}
+
+.col-fourth {
+  flex-basis: 25%;
+}
+
+.header {
+  padding: 1rem 4rem 0.5rem;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#top-logo {
+  width: 100px;
+}
+
+.nav-bar {
+  display: flex;
+  align-items: center;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  color: #a4a4a4;
+  font-size: 13px;
+  padding: 0 1.5rem;
+  width: 400px;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+button {
+  color: white;
+  background-color: #294464;
+  border: none;
+  border-radius: 3px;
+}
+
+.contact-button {
+  padding: 0.7rem 0.8rem;
+}
+
+hr {
+  background-color: #edeef0;
+  height: 1px;
+  border: none;
+}
+
+.hero {
+  background-image:
+    linear-gradient(rgba(105, 105, 105, 0.4), rgba(105, 105, 105, 0.4)),
+    url('images/hero.jpg');
+  background-size: cover;
+  background-position: center;
+  color: white;
+  height: 450px;
+  margin: 3rem 4rem;
+  padding: 4rem 3rem;
+}
+
+h1,
+h2 {
+  font-family: Tiempos Headline, serif;
+  font-weight: normal;
+  margin: 0;
+}
+
+p {
+  font-size: 13px;
+  line-height: 1.4rem;
+  margin: 0;
+}
+
+.hero-heading,
+.hero-par {
+  padding-right: 46%;
```

Comment:
That's a lot of padding!! Why is this necessary? There is likely a better way to accomplish this.